hack/jenkins/windows_integration_test_docker.ps1

```diff
@@ -15,6 +15,8 @@
 mkdir -p out
 
 (New-Object Net.WebClient).DownloadFile("https://dl.k8s.io/release/v1.20.0/bin/windows/amd64/kubectl.exe", "C:\Program Files\Docker\Docker\resources\bin\kubectl.exe")
```

Comment:
not related to this PR, but i feel like took installation should happen in common.ps1 instead of here

Reply:
Forgot to change this after rebasing onto common.ps1 PR. Also added comment explaining why we download kubectl.